x/programs/cmd/simulator/build.rs

```diff
@@ -24,7 +24,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let current_dir = std::env::current_dir().unwrap();
 
-    let simulator_path = "bin/simulator";
+    let simulator_path = "../../v2/bin/simulator";
```

Comment:
Are we leaving this as-is?

Reply:
For now. Step 1, get tests working again. Step 2, rename.

Reply:
David took care of moving things back from the `v2` directory in this PR, so this addresses that!